pagopa/pn-frontend-e2e
# ref/9162 - control on altri recapiti fields

## Short description

added check on table "GIà associati", moved a duplicated method, added new method to insert records in the table

## List of changes proposed in this pull request

- implemented steps to insert record in the table "GIà associati" and checks on the table's fields
- moved a method in the correct file
- updated a test with the correct method (from PF to PG)

## How to test

Simply run `mvn clean test -Dcucumber.filter.tags=@TA_visualizzazioneSezioneAltriRecapitiPG` and verify if the test works fine.
